### PR TITLE
Fix ownership of aodh dirs

### PR DESCRIPTION
**Pull request by pkilambi**

### CODE REVIEW

File: openstack-aodh.spec [hunk 1]
```diff
@@ -260,15 +260,17 @@ exit 0
...
 
 %files common
+%doc README.rst
 %dir %{_sysconfdir}/aodh
 %config(noreplace) %attr(-, root, aodh) %{_sysconfdir}/aodh/aodh.conf
 %config(noreplace) %attr(-, root, aodh) %{_sysconfdir}/aodh/policy.json
 %config(noreplace) %attr(-, root, aodh) %{_sysconfdir}/aodh/api_paste.ini
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
-%dir %{_localstatedir}/log/aodh
+%dir %attr(0755, aodh, root)  %{_localstatedir}/log/aodh
```
Comment (EmilienM) [inline openstack-aodh.spec]: no, it should be root:aodh or aodh:aodh I think

File: openstack-aodh.spec [hunk 2]
```diff
@@ -260,15 +260,17 @@ exit 0
 
 
 %files common
+%doc README.rst
```
Comment (EmilienM) [inline openstack-aodh.spec]: this is out of this patch context, isn't?

File: openstack-aodh.spec (hunk 1 above)
Comment (pkilambi) [inline openstack-aodh.spec]: hmm i dont think so. aodh:root is correct for a dir. I can confirm we do the same for other packages.

Comment (EmilienM) [inline openstack-aodh.spec]: ok, wfm if you tested it :)

File: openstack-aodh.spec (hunk 2 above)
Comment (pkilambi) [inline openstack-aodh.spec]: i just moved it from few lines below for clarity, its not new.

Comment (EmilienM) [inline openstack-aodh.spec]: ok, fair enough

**Comment by EmilienM** [comment]
:+1:  for merging

**Comment by hguemar** [comment]
Thanks gentlemen ! :tophat:

**Pull request merged**